motr/io.c

```diff
@@ -556,6 +556,7 @@ static int obj_io_init(struct m0_obj      *obj,
 	ioo->ioo_sns_state = SRS_UNINITIALIZED;
 	ioo->ioo_ext = *ext;
 	ioo->ioo_flags = flags;
+	ioo->ioo_flags |= M0_OOF_SYNC;
```

Comment:
We want to turn this flag on in all cased?

Reply:
Yes, as no perf impact is seen in preliminary testing.
Also this will avoid S3 changes as well for now.

Reply:
OK